src/pages/settings/Subscription/TaxExemptActions/index.tsx

```diff
@@ -37,27 +35,15 @@ function TaxExemptActions() {
         [translate],
     );
 
-    const calculateAndSetThreeDotsMenuPosition = useCallback(() => {
-        if (shouldUseNarrowLayout) {
-            return Promise.resolve({horizontal: 0, vertical: 0});
-        }
-        return new Promise<AnchorPosition>((resolve) => {
-            threeDotsMenuContainerRef.current?.measureInWindow((x, y, width, height) => {
-                resolve({
-                    horizontal: x + width,
-                    vertical: y + height,
-                });
-            });
-        });
-    }, [shouldUseNarrowLayout]);
+    const {calculatePopoverPosition} = usePopoverPosition();
 
     return (
         <View
             ref={threeDotsMenuContainerRef}
             style={[styles.mtn2, styles.pAbsolute, styles.rn3]}
         >
             <ThreeDotsMenu
-                getAnchorPosition={calculateAndSetThreeDotsMenuPosition}
+                getAnchorPosition={() => calculatePopoverPosition(threeDotsMenuContainerRef, anchorAlignment)}
```

Comment:
Should we wrap this with useCallback (and in other places too). What do you think?